libsupport/include/katana/EntityTypeManager.h

```diff
@@ -22,7 +23,7 @@ namespace katana {
 /// EntityTypeID is represented using 8 bits
 using EntityTypeID = uint8_t;
 static constexpr EntityTypeID kUnknownEntityType = EntityTypeID{0};
-static constexpr std::string_view kUnknownEntityTypeName = "kUnknownName";
+static constexpr std::string_view kUnknownEntityTypeName = "UnknownName";
```

Comment:
nit: I think there has been some discussion about what this string should be (or if there should be a string here at all). I would like to propose "Unknown Type". I think it is more descriptive and I don't think we need to restrict ourselves to CamelCase with these names.